tests/OSPSuite.Presentation.Tests/Presentation/HeavyWorkManagerSpecs.cs

```diff
@@ -133,5 +135,43 @@ public void should_dispose_of_the_presenter_when_the_action_is_completed()
       {
          A.CallTo(() => _heavyWorkPresenter.Dispose()).MustHaveHappened();
       }
+
+      public class When_the_heavy_work_manager_is_starting_an_action : concern_for_HeavyWorkManager
+      {
+         private Action _action;
```

Comment:
This is never assigned. You can use an anonymous action inline instead `() => { }`